tests/unit/src/unit/issues/Issue3345.hx

```diff
@@ -8,14 +8,14 @@ private abstract Meters(Float) from Float {
 		this = f;
 	}
 
-	@:op(-A) extern public static function neg(s:Meters):Meters;
-	@:op(A+B) extern public static function add(lhs:Meters, rhs:Meters):Meters;
-	@:op(A-B) extern public static function sub(lhs:Meters, rhs:Meters):Meters;
-	@:op(A>B) extern public static function gt(lhs:Meters, rhs:Meters):Bool;
-	@:op(A>=B) extern public static function gte(lhs:Meters, rhs:Meters):Bool;
-	@:op(A<B) extern public static function lt(lhs:Meters, rhs:Meters):Bool;
-	@:op(A<=B) extern public static function lte(lhs:Meters, rhs:Meters):Bool;
-	@:op(A==B) extern public static function eq(lhs:Meters, rhs:Meters):Bool;
+	@:op(-A) static private function neg(s:Meters):Meters;
```

Comment:
Let's not touch visibility.

Reply:
It does not concern the test (so it passes) and it has erroneous `extern` anyway, why not change it now?